pkg/controller/controller_test.go

```diff
@@ -497,83 +509,105 @@ func (s *ControllerSuite) TestSynchronousFailure(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *ControllerSuite) TestNilPrometheusRegistry(c *C) {
+	config, _ := kube.LoadConfig()
```

Comment:
Good practice to catch the error whenever possible. 
```suggestion
	config, err := kube.LoadConfig()
	c.Assert(err, IsNil)
```

Reply:
Thank you for catching this. I've also added an additional assert for `config` variable as well.